src/plugins/geomap/demo/config-fr.js

```diff
@@ -154,45 +163,62 @@ var wet_boew_geomap = {
 		},
 		{
 			title: "TopoJSON (World 110m)",
-			caption: "*NEEDS TRANSLATION*This is a sample dataset loaded from a remote TopoJSON resource.",
+			caption: "Voici un exemple de jeu de données chargé à partir d'une ressource TopoJSON distante.",
 			type: "topojson",
 			url: "demo/topojson.json",
 			accessible: false,
+			visible: false,
 			style: {
 				strokeColor: "#3399ff",
 				strokeWidth: 2.0
+			}
+		},
+		{
+			title: "Esri REST Tile Layer",
+			caption: "Il s'agit d'un exemple de jeu de données chargé à partir d'un service de tuiles REST Esri distant.",
+			type: "esritile",
+			url: "//maps-cartes.services.geo.ca/server_serveur/rest/services/NRCan/Carte_climatique_HOT2000_Climate_Map_EN/MapServer/",
+			params: {
+				LAYERS: "show:0"
 			},
-			visible: false
+			visible: false,
+			datatable: false,
+			options: {
+				legendHTML: "<ul class='list-unstyled'>" +
+					"<li><small>Weather Station</small><img src='data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAAAXNSR0IB2cksfwAAAAlwSFlzAAAOxAAADsQBlSsOGwAAAFZJREFUOI3t0TEKwDAMQ1F9yIFzlN5YHUqgpOA2TZaANdnLQ9hFi1MSTPCKbbcZYAq0bWq3B2gI9pgkUWN086cAPG54xI95bdjQ+/674VdkGBxJgvM5AZAOH6jK5pnSAAAAAElFTkSuQmCC'></li>" +
+					"</ul>"
+			}
 		},
 		{
 			title: "EsriJSON",
-			caption: "*NEEDS TRANSLATION*This is a sample dataset loaded from a remote Esri JSON resource.",
+			caption: "Ceci est un exemple de jeu de données chargé à partir d'une ressource Esri JSON distante.",
 			type: "esrijson",
-			url: "//geoappext.nrcan.gc.ca/arcgis/rest/services/NEB/Pipeline/MapServer/0/query?where=OBJECTID>0&f=pjson",
+			url: "https://geoappext.nrcan.gc.ca/arcgis/rest/services/FGP/TMX_EN/MapServer/2/query?where=OBJECTID>0&f=pjson",
 			attributes: {
-				OPERATOR: "Opérateur"
+				OPERATOR: "Operator"
```

Comment:
Can we keep the French term "Opérateur"?

Reply:
Fixed